packages/next/server/web/spec-extension/fetch-event.ts

```diff
@@ -10,13 +10,22 @@ export class NextFetchEvent extends FetchEvent {
     this.sourcePage = params.page
   }
 
-  // @ts-ignore
+  /**
+   * @deprecated The `request` is now the first parameter and the API is now async.
+   *
+   * Read more: https://nextjs.org/docs/messages/middleware-new-signature.
```

Comment:
```suggestion
   * Read more: https://nextjs.org/docs/messages/middleware-new-signature
```
removing period as this is included in the URL in some IDE/terminals